modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/commands/openTagSelectionModal.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {addParams, navigate, openSelectionModal} from 'frontend-js-web';
+
+type Tag = {
+	qaId: string;
+	selectable: boolean;
+	value: string;
+};
+
+export default function openTagSelectionModal({
```

Comment:
These two utilities should be in `frontend-js-web`, [siblings to the likes of openAlertModal and openConfirmModal](https://github.com/liferay/liferay-portal/tree/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/commands). `frontend-js-components-web` is for common React components.

Please ignore inconsistencies in format of file names :sweat_smile: The camelCase, like in this PR, is the right format.

Reply:
@markocikos thanks for the review! I have updated the PR with the change of the module you suggested. Finally, I added the code with a `.js` extension since in this case, `{addParams, navigate, openSelectionModal}` are imported in these files and they are functions that have not yet been converted to typescript, so they couldn't be used in a `.ts` file (see https://github.com/liferay-frontend/liferay-portal/pull/3776/commits/1df359a31ae4d729ecfd9bec2e70bf2329159ad9 and https://github.com/liferay-frontend/liferay-portal/pull/3776/commits/1ba2e89b0e37dd3264d9a7c165c0be4d512cacc5).